linetools/utils.py

```diff
@@ -212,10 +216,13 @@ def radec_to_coord(radec):
             else:
                 DEC = radec[1]
             #
-            coord = SkyCoord(radec[0]+DEC, frame='icrs',
+            coord = SkyCoord(radec[0]+DEC, frame=frame,
```

Comment:
this may give the wrong coordinate if `frame=='galactic'`... although not sure.

Reply:
i think we are ok here.